tests/e2e-tests/specs/backend/reviews-by-category.test.js

```diff
@@ -0,0 +1,48 @@
+/**
+ * External dependencies
+ */
+import { switchUserToAdmin, clickButton } from '@wordpress/e2e-test-utils';
+import { visitBlockPage } from '@woocommerce/blocks-test-utils';
+
+const block = {
+	name: 'Reviews by Category',
+	slug: 'woocommerce/reviews-by-category',
+	class: '.wc-block-reviews-by-category',
+};
+
+describe( `${ block.name } Block`, () => {
+	beforeAll( async () => {
+		await switchUserToAdmin();
+		await visitBlockPage( `${ block.name } Block` );
+	} );
+
+	it( 'renders without crashing', async () => {
+		await expect( page ).toRenderBlock( block );
+	} );
+
+	it( 'shows category selector', async () => {
+		await expect( page ).toMatchElement(
+			`${ block.class } .woocommerce-search-list`
+		);
+	} );
+
+	it( 'can select a category and show reviews', async () => {
```

Comment:
I couldn't get this test to pass (tried running twice):

```
 FAIL  tests/e2e-tests/specs/backend/reviews-by-product.test.js (35.76s)
  Reviews by Product Block
    ✓ renders without crashing (190ms)
    ✓ shows product selector (6ms)
    ✕ can select a product and show reviews (30019ms)

  ● Reviews by Category Block › can select a category and show reviews

    : Timeout - Async callback was not invoked within the 30000ms timeout specified by jest.setTimeout.Timeout - Async callback was not invoked within the 30000ms timeout specified by jest.setTimeout.Error:
```

Ran in dev mode (`npm run test:e2e-dev`) and I think I saw the problem - no reviews on the selected product:

<img width="1053" alt="Screen Shot 2020-07-03 at 10 42 44 AM" src="https://user-images.githubusercontent.com/4167300/86415084-02ae5400-bd1a-11ea-845a-1314bfecdbe7.png">

Reply:
D'oh! I needed to `wp-env clean` to get the new fixture data. We can probably make this clearer in the docs. 

Over time it'll become clearer if the fixture data is slower-moving than the tests themselves – if they are closely coupled maybe we can change the move the data setup to the start of the test command, so `npm run test:e2e` re-applies the fixture data and runs tests.

Reply:
So this is good to go :)